ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1840,6 +1840,36 @@ tests:
     test:
     - chain: openshift-e2e-test-mco-qe-longduration
     workflow: cucushift-installer-rehearse-gcp-ipi
+- as: gcp-ipi-f9-mco-techpreview-part1
```

Comment:
We should name the job using the `longduration` word in the name and `g1` instead of `part1`, so that we name the job the same way the existing jobs are already named

It looks like `tp` is the way existing jobs are naming techpreview jobs, so we can use it too.

Existing job:
`gcp-ipi-f9-longduration-mco-g1`

We should probably name it like:
`gcp-ipi-f9-longduration-tp-mco-g1`